lib/potluck.rb

```diff
@@ -0,0 +1,37 @@
+require './lib/dish.rb'
+require 'pry'
+
+class Potluck
+
+  attr_accessor :date,
+                :dishes,
+                :name,
+                :category,
+                :dishes_by_category
+
+  def initialize(date)
+    @date = date
+    @dish = Dish.new(name, category)
+    @name = name
+    @category = category
+    @dishes = []
+    @dishes_by_category = Hash.new([])
```

Comment:
Careful with this. Think that you really only need `@date` and `@dishes` instance variables.